app/server/Main.hs

```diff
@@ -70,11 +72,15 @@ main = do
     network <- HttpBridge.newNetworkLayer (T.pack . encode $ networkName) bridgePort
     db <- MVar.newDBLayer
     let wallet = mkWalletLayer db network
-    wid <- createWallet wallet NewWallet
-        { mnemonic = mnemonicSentence
-        , mnemonic2ndFactor = mempty
-        , name = WalletName "My Wallet"
-        , passphrase = mempty
-        , gap = minBound
-        }
-    watchWallet wallet wid
+    widE <- runExceptT $ createWallet wallet NewWallet
```

Comment:
why have we decided to start a wallet server with a default wallet compared to starting a wallet server without one where wallet is created on demand by calling wallet server endpoint ?

Reply:
cannot answer here -> just refactor this snippet just in order to compile

Reply:
@Anviking there's no default wallet. Instead, there's just a very basic CLI that will "monitor" a wallet associated to some mnemonic that are given. This is merely for testing and to be able to play with our code instead of waiting 4 months to actually integrate with anything. 

There's no web server at the moment, and, although we'll probably implement the beginning of this in the upcoming days, at this stage, it's still quite useful to be able to connect all the pieces together and already perform some tests to make sure that we are doing something sensible with the primitives.

The `server/Main.hs` is really not in its final stage.